tests/test_utils.py

```diff
@@ -151,7 +154,12 @@ def validate_uuid4(uuid_string: str) -> bool:
 
 
 def get_sqlite_temp_table_names(engine):
-    result = engine.execute(
+    if isinstance(engine, Engine):
+        connection: Connection = engine.connect()
```

Comment:
Same comment about putting this in a with block to ensure the connection is closed